### ISSUE TITLE
More specific width tag handling

### ISSUE THREAD

**Issue by Trietes** [valhalla/valhalla#4940]
What do you guys think about treating other widht related tags similar as the `maxwidth` tag?
- The tag `maxwidth:physical` is about the physical width limit of a street. I think it should be treated exactly like the legally allowed width. Adding it to the graph.lua should also not be a big issue.
- I'm unsure about the `widht` tag. It feels bad routing over a road which is tagged as having a width smaller than the vehicles width. But the definition of the tag seems to be not that clear (https://wiki.openstreetmap.org/wiki/Key:width)

**Comment by chrstnbwnkl** [comment]
`maxwidth:physical` has very little coverage (total of ~800 ways worldwide). And I'm not sure if it should be treated the same way as `maxwidth`, because if you have a road with a given physical width of 3m with walls to the right and left, would you let a vehicle with a width of 2.99 pass? Seems a bit adventurous :smile: 

And the width tag is not well defined for roads, it could mean the width of the whole road instead of just one lane, so I don't think it makes sense to parse it and treat it like a limit for vehicles. But happy to hear other opinions.

**Comment by Trietes** [comment]
For `maxwidth:physical` I would say its better to handle it as a limit than not handling at all. Even though that would mean routing a 2.99 meter vehicle through a 3m width road. The low coverage is ofc an argument.

**Comment by kevinkreiser** [comment]
> its better to handle it as a limit than not handling at all

this is true and it doesnt hurt us to add it to the lua processing, cant be that much code even if it is only covering a few ways world wide

**Comment by Trietes** [comment]
I've just seen that graph.lua already handles the maxwidth:physical tag here: https://github.com/valhalla/valhalla/blob/678834f0add5fa54347cad59abc0818910f32b26/lua/graph.lua#L1814. So I'll close this one
